app/knowledge_panels.py

```diff
@@ -39,3 +41,59 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def last_edits_kp(
+    facet: str,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    if facet == "packaging":
+        plural = facet
+    else:
+        plural = facet_plural(facet=facet)
+    query = {
+        "fields": "code,last_editor,last_modified_t",
+        "sort_by": "last_modified_t",
+    }
+    if facet == "country":
+        country_code = country_to_ISO_code(value=value)
+        country = value
+        url = f"https://{country_code}-en.openfoodfacts.org"
+        description = f"{value}"
```

Comment:
a good strategy with code, is to unify then process.

If you look at what we did for the first facet, if facet == "country" can be unified to having a country parameter and no facet.

Although it's not mandatory, It can be good exercice for you to try to do the same here.